package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/inteli-api",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Insert repo description",
   "main": "app/index.js",
   "scripts": {
```

Comment:
do we need to define a migration script here?

Reply:
something like `"migrate": "knex migrate:latest"`? That would be nice, could add in next PR to save re-review here